src/rules/orderedImportsRule.ts

```diff
@@ -51,16 +60,23 @@ export class Rule extends Lint.Rules.AbstractRule {
                     type: "string",
                     enum: ["case-insensitive", "lowercase-first", "lowercase-last"],
                 },
+                "source-imports-order": {
+                    type: "string",
+                    enum: ["case-insensitive", "lowercase-first", "lowercase-last"],
+                },
             },
             additionalProperties: false,
         },
-        optionExamples: ["true", '[true, {"named-imports-order": "lowercase-first"}]'],
+        optionExamples: [
+            "true",
+            '[true, {"named-imports-order": "lowercase-first", "source-imports-order": "lowercase-last"}]',
```

Comment:
although it's good form to name all local files in that way as well to avoid filesystem case issues and the like,